pkg/cgroups/cgroups.go

```diff
@@ -525,9 +525,12 @@ func (c *CgroupControl) AddPid(pid int) error {
 // Stat returns usage statistics for the cgroup
 func (c *CgroupControl) Stat() (*Metrics, error) {
 	m := Metrics{}
-	for _, h := range handlers {
+	for name, h := range handlers {
 		if err := h.Stat(c, &m); err != nil {
-			return nil, err
+			if !os.IsNotExist(errors.Cause(err)) {
+				return nil, err
+			}
+			logrus.Warningf("failed to get %s stats: %v", name, err)
```

Comment:
```suggestion
			logrus.Warningf("Failed to Cgroup stats: %v", err)
```
Assuming the name is already in the error message